packages/api/resolvers/mutations/updateCartItem.js

```diff
@@ -26,27 +26,8 @@ export default function updateCartItem(
     throw new OrderWrongStatusError({ status: order.status });
   }
 
-  if (quantity !== null) {
-    if (quantity < 1) throw new OrderQuantityTooLowError({ quantity });
-    // FIXME: positionId is actually
-    OrderPositions.updatePosition(
-      {
-        orderId: item.orderId,
-        positionId: itemId,
-      },
-      { quantity }
-    );
-  }
-
-  if (configuration !== null) {
-    OrderPositions.updatePosition(
-      {
-        orderId: item.orderId,
-        positionId: itemId,
-      },
-      { configuration }
-    );
-  }
+  if (quantity && quantity < 1)
```

Comment:
This will lead to quantity 0 not throwing the exception, which is not what you want

Reply:
quantity 0 will short circuit on the first test since 0 is false.